src/commands/session.ts

```
@@ -10,21 +12,175 @@ import {
+	args: {
+		...sharedCommandConfig.args,
+		id: {
+			type: 'string',
+			short: 'i',
+			description: 'Load usage data for a specific session ID',
+		},
+	},
+	toKebab: true,
 	async run(ctx) {
 		// --jq implies --json
 		const useJson = ctx.values.json || ctx.values.jq != null;
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)